packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/java/enrollmentqrcodeintegrationsteps.md

```diff
@@ -0,0 +1,51 @@
+#### 1. Initiate sign-in flow and return a list of authenticators
+
+The user signs in with a username and password, and then chooses Okta Verify from a list of authenticators. This is covered in the earlier [Shared Code](#initiate-sign-in-and-return-a-list-of-authenticators) section.
+
+#### 2. Request QR code
+
+The user must select the Okta Verify Authenticator, using the common `selectAuthenticator()` method described above, then passing the result into `idxAuthenticationWrapper.selectAuthenticator()`.
+
+
+```java
+case AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION:
+    return idxAuthenticationWrapper.selectAuthenticator(proceedContext,
+        selectAuthenticator(authenticationResponse));
+```
+
+#### 3: Display QR code and start polling
+
+When the user selects to enroll the Okta Verify `Authenticator` the form posts back to the `idxAuthenticationWrapper.selectAuthenticator()` method. This will trigger an `AWAITING_POLL_ENROLLMENT` state.  The state's `ContextualData` object contains a base64 encoded QR code image that can be displayed to the user.
```

Comment:
```suggestion
When the user selects to enroll the Okta Verify `Authenticator`, the form posts back to the `idxAuthenticationWrapper.selectAuthenticator()` method. This will trigger an `AWAITING_POLL_ENROLLMENT` state. The state's `ContextualData` object contains a base64 encoded QR code image that can be displayed to the user.
```